jenkins-integration/beaker/install/shared/23_install_system_gems.rb

```diff
@@ -0,0 +1,17 @@
+require 'json'
+
+test_name 'Install System Gems'
+
+def install_system_gem(host, gem_name)
+  step "Install '#{gem_name}' gem"
+  # Ensure it's the system ruby
+  on(host, "/usr/bin/env gem install #{gem_name}")
+end
+
+step 'Install gems using system ruby'
+
+gem_list = ENV['PUPPET_GATLING_SYSTEM_GEMS'].split(',')
```

Comment:
I don't want to introduce gems into the `Jenkinsfile` layer until we have a forcing function.  I'd be fine with changing the data structure that we use here such that it would support (require?) specific gem versions to be specified in the beaker setup code, but it shouldn't bubble all the way up to the Jenkinsfile for now.